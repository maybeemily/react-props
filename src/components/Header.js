import React from 'react';

export default function Header() {
  return (
    <>
    <img src="../../assets/saturday.jpg"></img>
    <h1>THE COOLEST DOG</h1>
    </>
  );
}
