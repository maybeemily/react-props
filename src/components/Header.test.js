import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('header test', () => {
  it('renders the header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
