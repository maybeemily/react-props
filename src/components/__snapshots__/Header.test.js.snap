// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header test renders the header 1`] = `
<Fragment>
  <img
    src="../../assets/saturday.jpg"
  />
  <h1>
    THE COOLEST DOG
  </h1>
</Fragment>
`;
